.prow/provider-vsphere.yaml

```diff
@@ -103,7 +103,7 @@ presubmits:
             limits:
               memory: 4Gi
 
-  - name: pre-kubermatic-e2e-vsphere-ubuntu-1.23-datastore-cluster
+  - name: pre-kubermatic-e2e-vsphere-ubuntu-1.24-datastore-cluster
```

Comment:
Please change the Kubernetes version for this job.

Reply:
Good catch!